gitlab_users

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
+from lib import get_gitlab_instance
+
+import sys
+
+SHOW_CONFIG = len(sys.argv) >= 2 and sys.argv[1] == 'config'
+
+if SHOW_CONFIG:
+    print('graph_title GitLab users')
+    print('graph_vlabel users')
+    print('graph_args -l 0')
+    print('graph_category gitlab')
```

Comment:
why? `sys.exit(0)` will force the script to exit and return exit code 0. We need the rest of the code to get the graph initialization right, because it's retrieved dynamically.